azure-pipelines.yml

```
@@ -2,185 +2,19 @@ trigger:
-    - script: >
-        set -o pipefail &&
-        xcodebuild $XCODE_FLAGS build -sdk watchsimulator -destination "name=Apple Watch Series 3 - 38mm"
-      displayName: build on watchOS
-      condition: succeededOrFailed()
-    - script: >
-        set -o pipefail &&
-        swift test --parallel
-      displayName: swift test with UTF16
-      condition: succeededOrFailed()
-      env:
-        YAMS_DEFAULT_ENCODING: UTF16
-    - script: >
-        set -o pipefail &&
-        swift test --parallel
-      displayName: swift test with UTF8
-      condition: succeededOrFailed()
-      env:
-        YAMS_DEFAULT_ENCODING: UTF8
-    - task: PublishTestResults@2
-      inputs:
-        testRunner: JUnit
-        testResultsFiles: build/reports/**
-      condition: succeededOrFailed()
-
-- job: CocoaPods
-  pool:
-    vmImage: 'macOS 10.13'
-  variables:
-    DEVELOPER_DIR: /Applications/Xcode_9.4.1.app
-  steps:
-    - script: pod lib lint
-      displayName: pod lib lint
-
-- job: Analyze
-  pool:
-    vmImage: 'macOS 10.13'
-  variables:
-    DEVELOPER_DIR: /Applications/Xcode_10.1.app
-  steps:
-    - script: xcodebuild -scheme Yams clean build-for-testing > xcodebuild.log
-      displayName: Generate xcodebuild.log
-    - script: HOMEBREW_NO_AUTO_UPDATE=1 brew install https://raw.github.com/Homebrew/homebrew-core/master/Formula/swiftlint.rb
-      displayName: Install SwiftLint
-    - script: swiftlint analyze --strict --compiler-log-path xcodebuild.log
-      displayName: Run SwiftLint Analyze
-
-- job: jazzy
-  pool:
-    vmImage: 'Ubuntu 16.04'
-  container: norionomura/jazzy:0.9.4_swift-4.2.1
-  steps:
-    - script: bundle install --path vendor/bundle
-      displayName: bundle install
-    - script: swift build
-      displayName: swift build
-    - script: sourcekitten doc --spm-module Yams > yams.json
-      displayName: Generate documentation json
-    - script: bundle exec jazzy --clean --sourcekitten-sourcefile yams.json
-      displayName: Run jazzy
-    - script: >
-        if ruby -rjson -e "j = JSON.parse(File.read('docs/undocumented.json')); exit j['warnings'].length != 0"; then
-          echo "Undocumented declarations:"
-          cat docs/undocumented.json
-          exit 1
-        fi
-      displayName: Validate documentation coverage
-    - task: PublishPipelineArtifact@0
-      inputs:
-        artifactName: 'API Docs'
-        targetPath: 'docs'
+    - powershell: Invoke-WebRequest 'https://compnerd.visualstudio.com/3133d6ab-80a8-4996-ac4f-03df25cd3224/_apis/build/builds/109/artifacts?artifactName=toolchain&api-version=5.0&%24format=zip' -outfile .\toolchain.zip
+      displayName: Download Swift toolchain for Windows
+    - powershell: Expand-Archive toolchain.zip -DestinationPath C:\Swift
+      displayName: Install Swift
+    - powershell: Write-Host "##vso[task.setvariable variable=PATH]$PATH;C:\\Swift\\toolchain\\Developer\\Toolchains\\unknown-Asserts-development.xctoolchain\\usr\\bin"
```

powershell does not seem to recognize `$PATH`.
```suggestion
    - powershell: Write-Host "##vso[task.setvariable variable=PATH]%PATH%;C:\\Swift\\toolchain\\Developer\\Toolchains\\unknown-Asserts-development.xctoolchain\\usr\\bin"
```